### PR TITLE
add documentation for POST /opengraph

### PR DESCRIPTION
**Pull request by cpanato**
code pr: https://github.com/mattermost/platform/pull/6343

### CODE REVIEW

File: v4/source/opengraph.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+  /opengraph:
+    post:
+      tags:
+        - opengraph
+      summary: Get Open Graph Metadata for a specif URL
```
Comment (jwilander) [inline v4/source/opengraph.yaml]: Can we change this to `Get open graph metadata for url`?

File: v4/source/opengraph.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      description: |
+        Get Open Graph Metadata for a specif URL.
```
Comment (jwilander) [inline v4/source/opengraph.yaml]: And this to `Use the Open Graph protocol to get some generic metadata about a URL. Used for creating link previews.`

File: v4/source/opengraph.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        ##### Permissions
+        No permission required.
```
Comment (jwilander) [inline v4/source/opengraph.yaml]: I think you need to be logged in right?

File: v4/source/opengraph.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/opengraph.yaml]: done

File: v4/source/opengraph.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/opengraph.yaml]: done, hope put the `Minimum`in the right place

File: v4/source/opengraph.yaml (hunk 3 above)
Comment (cpanato) [inline v4/source/opengraph.yaml]: done

**Pull request merged**